src/client/routes/manage/SliderInput.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import { RadioSlider } from '../../components/Buttons/RadioSlider';
+import { QueriedHacker } from './HackerTableTypes';
+import { ApplicationStatus, HackerStatusesMutationFn } from '../../generated/graphql';
+
+export interface DeselectElement extends HTMLDivElement {
+	context: {
+		selectable: {
+			clearSelection: () => void;
+		};
+	};
+}
+
+// maps the radio slider labels to the hacker status
+export const processSliderInput = (input: string): ApplicationStatus => {
+	switch (input.toLowerCase()) {
+		case 'accept':
+			return ApplicationStatus.Accepted;
+		case 'reject':
+			return ApplicationStatus.Rejected;
+		case 'undecided':
+		default:
+			return ApplicationStatus.Submitted;
+	}
+};
+
+interface SliderInputProps {
+	deselect: React.RefObject<DeselectElement>;
+	selectedRowsEmails: string[];
+	sortBy: keyof QueriedHacker;
```

Comment:
```suggestion
	sortBy?: keyof QueriedHacker;
```
Fix linter error on HackerTable.tsx(367,11)